lib/Terms/CreateTranslation.php

```diff
@@ -83,11 +84,14 @@ public function redirect_to_new( int $term_id, int $tt_id, string $taxonomy, boo
 		// TODO: Add something in the page to show that a translation is being saved. Use existence of ubb_source.
 		wp_safe_redirect(
 			add_query_arg(
-				[
-					'taxonomy'   => $taxonomy,
-					'lang'       => $lang_create,
-					'ubb_source' => $term_id,
-				],
+				array_filter(
+					[
+						'taxonomy'   => $taxonomy,
+						'lang'       => $lang_create,
+						'ubb_source' => $term_id,
+						'post_type'  => $_REQUEST['ubb_post_type'] ?? '',
```

Comment:
We don't need the nullish coalescence, because `null` will be caught by the filter.

Reply:
It returns null but throws a warning.